src/store/utils.js

```diff
@@ -66,8 +67,21 @@ export const getLegacyRskBalance = async (accounts) => {
     .addProvider(
       new EthereumRpcProvider({ uri: 'https://public-node.rsk.co' })
     )
+    
+  if(mnemonic) {
```

Comment:
At this point, why aren't we using `createClient` function? There's too much duplication here

Reply:
Because createClient is used to set the new or old derivation path and this one is used always to log the user with the old derivation path and check the balance and base on that to determine if createClient() should use new or old derivation path